### PR TITLE
Rebase and correct styles on #3763

### PR DESCRIPTION
**Pull request by indirect**

### CODE REVIEW

**Comment by indirect** [comment]
@homu r+

**Comment by segiddins** [comment]
@indirect failures look legit?

**Comment by neoeno** [comment]
They do. I'll try to get to the bottom of it this weekend (unless someone else fancies it). Any clues on what could be the significant difference in older versions of RubyGems?

File: spec/commands/update_spec.rb [hunk 1]
```diff
@@ -109,6 +129,96 @@
...
+        source "file://#{gem_repo2}"
+        gem "activesupport"
+      G
+      update_repo2 { build_gem "activesupport", "3.0" }
+
+      bundle "update --source activesupport"
+      should_not_be_installed "activesupport 3.0"
+    end
+
+    it "should update gems outwith the source that happen to have the same name" do
```
Comment (segiddins) [inline spec/commands/update_spec.rb]: `outwith` ?

Comment (neoeno) [inline spec/commands/update_spec.rb]: Ah, sorry — Scottish English. In this context it means 'gems not sourced from the provided source'. I'll amend for clarity (hopefully more clear than the example I gave).

Comment (segiddins) [inline spec/commands/update_spec.rb]: no worries, just confused me :)

**Comment by indirect** [comment]
Closing in favor of #3763.

**Pull request closed without merging**